client/views/admin/apps/AppsRoute.tsx

```diff
@@ -39,17 +39,17 @@ const AppsRoute = () => {
 
 		initialize();
 
-		return () => {
+		return (): void => {
 			mounted = false;
 		};
 	}, [canViewAppsAndMarketplace, isAppsEngineEnabled, appsWhatIsItRoute]);
 
 	const context = useRouteParameter('context');
 
-	const isMarketPlace = !context;
+	const isMarketplace = !context;
 
 	const id = useRouteParameter('id');
-	const version = useRouteParameter('version');
+	// const version = useRouteParameter('version');
```

Comment:
Hm.. what was version being used for before?

Reply:
it was used to be passed to a inner component, but now we are performing `useRouteParameter('version')` there